boards/stm32f0discovery/dist/debug.sh

```diff
@@ -1,4 +1,19 @@
 #!/bin/sh
 
-echo "Debugging $1"
-arm-none-eabi-gdb -tui -command=$1 $2
+openocd -f "board/stm32f0discovery.cfg" \
+    -c "tcl_port 6333" \
+    -c "telnet_port 4444" \
+    -c "init" \
```

Comment:
To keep things somewhat synchronised you could add something like [here](https://github.com/RIOT-OS/RIOT/pull/2364/files#diff-e0996e5f8c0e4adf96a4172c54084dc5R7)